From c76085303111df3e88e69cec42956a7da85a484e Mon Sep 17 00:00:00 2001
From: Aleksandr Slobodeniuk <aslobodeniuk@fluendo.com>
Date: Wed, 20 Dec 2017 12:27:21 +0100
Subject: [PATCH] Adapting patch to v1.6.1:

> From 9f724884ab61061e55b9350267fb609d172114d8 Mon Sep 17 00:00:00 2001
> From: Edward Hervey <bilboed@bilboed.com>
> Date: Sat, 14 Feb 2015 15:08:37 +0100
> Subject: [PATCH 4/6] build: Remove (broken) custom android toolchain usage

> Amazed no one complained earlier, it was always picking the arm
> toolchain.
---
 build/make/configure.sh | 28 ----------------------------
 1 file changed, 28 deletions(-)

diff --git a/build/make/configure.sh b/build/make/configure.sh
index 007e02000..c5f825d4c 100644
--- a/build/make/configure.sh
+++ b/build/make/configure.sh
@@ -981,34 +981,6 @@ EOF
         android*)
           if [ -n "${sdk_path}" ]; then
             SDK_PATH=${sdk_path}
-            COMPILER_LOCATION=`find "${SDK_PATH}" \
-              -name "arm-linux-androideabi-gcc*" -print -quit`
-            TOOLCHAIN_PATH=${COMPILER_LOCATION%/*}/arm-linux-androideabi-
-            CC=${TOOLCHAIN_PATH}gcc
-            CXX=${TOOLCHAIN_PATH}g++
-            AR=${TOOLCHAIN_PATH}ar
-            LD=${TOOLCHAIN_PATH}gcc
-            AS=${TOOLCHAIN_PATH}as
-            STRIP=${TOOLCHAIN_PATH}strip
-            NM=${TOOLCHAIN_PATH}nm
-
-            if [ -z "${alt_libc}" ]; then
-              alt_libc=`find "${SDK_PATH}" -name arch-arm -print | \
-                awk '{n = split($0,a,"/"); \
-                split(a[n-1],b,"-"); \
-                print $0 " " b[2]}' | \
-                sort -g -k 2 | \
-                awk '{ print $1 }' | tail -1`
-            fi
-
-            if [ -d "${alt_libc}" ]; then
-              add_cflags "--sysroot=${alt_libc}"
-              add_ldflags "--sysroot=${alt_libc}"
-            fi
-
-            # linker flag that routes around a CPU bug in some
-            # Cortex-A8 implementations (NDK Dev Guide)
-            add_ldflags "-Wl,--fix-cortex-a8"
 
             enable_feature pic
             soft_enable realtime_only
-- 
2.11.0

